src/defaults/collections.json

```diff
@@ -0,0 +1,49 @@
+[
```

Comment:
Maybe this file would be better of as a typescript file, the repetition of `bani` would be avoidable.